HonestHouse/HonestHouse/Sources/Core/Network/DigestAuth/NetworkManager.swift

```diff
@@ -103,8 +103,13 @@ final class NetworkManager {
                     }
                 }
                 
-                // 401이 아닌 응답은 그대로 반환
+                // 다른 http 에러 검증
+                try validateResponse(response)
+                
                 return response
+                
+            } catch let error as CCAPIError {
+                throw error
             } catch let error as MoyaError {
                 if case .statusCode(let response) = error, response.statusCode == 401 {
```

Comment:
```suggestion
            } catch let ccapiError as CCAPIError {
                throw ccapiError
            } catch let moyaError as MoyaError {
                if case .statusCode(let response) = moyaError, response.statusCode == 401 {
```

이렇게 바꿔도 좋을 것 같아요! 다른 에러들도 그냥 `error`말고 에러 이름으로 변수명을 가지고 있어서요

Reply:
다른 통일 안된 부분도 수정 완!